src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+body {
+  width: 48px;
+  height: 58px;
```

Comment:
The body dimensions are set to 48px by 58px, which might not be sufficient to display all the star blocks properly. Ensure that these dimensions are appropriate for your layout requirements.